pkg/cluster/config/validate.go

```diff
@@ -26,12 +26,53 @@ import (
 // with the config, or nil if there are none
 func (c *Config) Validate() error {
 	errs := []error{}
-	if c.Image == "" {
+
+	// All nodes in the config should be valid
+	for _, n := range c.nodes {
+		if err := n.Validate(); err != nil {
+			errs = append(errs, fmt.Errorf("Please fix invalid configuration for node %q: \n%v", n.Name, err))
```

Comment:
errors in this file seem to start with uppercase.
also we should use pkg/errors.

Reply:
fixed